src/codewhisperer/util/supplementalContext/crossFileContextUtil.ts

```diff
@@ -152,10 +153,29 @@ function splitFileToChunks(filePath: string, chunkSize: number): Chunk[] {
  * This function will return relevant cross files for the given editor file
  * by referencing open files, imported files and same package files.
  */
-async function getRelevantCrossFiles(editor: vscode.TextEditor, dependencyGraph: DependencyGraph): Promise<string[]> {
-    return getOpenFilesInWindow().filter(file => {
+export async function getRelevantCrossFiles(editor: vscode.TextEditor): Promise<string[]> {
+    // TODO: make sure this will work across OSs?
+    const fileSeperator = path.sep
```

Comment:
need to confirm this for windows

Reply:
Don't handle these by yourself. 

Look for existing util functions 

1. in this repo
2. By vs code
3. By nodejs

Reply:
yes but Ithought  `path.sep ` is a nodejs API, should be fine right?

Reply:
instead of doing something `windos ? '\' : '/'` manually

Reply:
may need your advice as well @JadenSimon @justinmk3

Reply:
There is almost never a reason to handle separators differently on Windows. `/` works fine unless you are invoking some old `cmd.exe` commands. Especially in the nodejs and vscode apis, `/` slashes work fine and should be preferred.  You may want to use normalize(): https://github.com/aws/aws-toolkit-vscode/blob/d66f115ee4e1fed8d87fa6a59bd83a1c5925ac05/src/shared/utilities/pathUtils.ts#L84-L91

Reply:
ack